.github/workflows/node.yaml

```
@@ -18,7 +18,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [6.x, 8.x, 10.x, 14.x]
+        node-version: [6.x, 8.x, 10.x, 14.x, 16.x]
```

```suggestion
        node-version: [6.x, 8.x, 10.x, 14.x]
```

Let's try without node 16 in the matrix.